src/services/resource/legalEntityManagementResource.ts

```diff
@@ -0,0 +1,14 @@
+import Client from "../../client";
+import Service from "../../service";
+import Resource from "../resource";
+
+class legalEntityManagementResource extends Resource {
```

Comment:
```suggestion
class LegalEntityManagementResource extends Resource {
```
Classnames should be capitalized